tool/generate_font.dart

```diff
@@ -34,7 +34,7 @@ void main(List<String> arguments) {
         String name = '${style}_$iconName';
         iconDefinitions[name] = generateIconDefinition(
           name,
-          styles.first,
+          style,
```

Comment:
Just to be sure -- this won't affect the current non-pro upgrade process?

Reply:
This is actually a fix for a bug I discovered when updating with the pro icons: The `style` of the for loop should be used here and not `styles.first` (this lead to only "bold" icons, even though their name was `light_something`, see line 34) - this only never surfaced because non-pro icons do not have more than two styles (bold, regular) and regular is removed before this loop.

So yes, everything is fine here

Reply:
I just did a test run to ensure the updater for non-pro icons works as intended and compared it to #43's icon list. There is a difference for one of the icons (namely brandsFontAwesomeLogoFull) where the current tool (before this fix) generates a type of IconDataSolid instead of IconDataBrands

(https://www.diffchecker.com/9mD3JFFA, left is current tool, right is updated with this fix)